docker/crawlclima/start_cron_celery.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+
 python docker/crawlclima/setup.py develop &
```

Comment:
I think you don't need this `&` at the end of line 2 (https://mywiki.wooledge.org/BashSheet#:~:text=An%20ampersand%20does%20the%20same,for%20the%20former%20to%20end.)